app/src/main/java/com/sdp/movemeet/Activity/Activity.java

```diff
@@ -71,7 +71,7 @@ public Activity(String activityId,
         if(activityId == null || organizerId == null || title == null || numberParticipant <= 0 )
             throw new IllegalArgumentException();
 
-        if(description == null || date == null || duration <= 0 || sport == null || address == null)
+        if(date == null || sport == null || address == null || duration <= 0 || description == null)
```

Comment:
Why did you change the order of the arguments in the if ?

Reply:
I tried to delete some fields during tests, and added it back again.